src/main/scala/com/monsanto/arch/cloudformation/model/simple/Builders.scala

```diff
@@ -244,6 +181,18 @@ trait Subnet extends AvailabilityZone with Outputs {
 
     f(sub) ++ sub.andOutput(name, name)
   }
+
+  def nat(routeTables: Seq[`AWS::EC2::RouteTable`])(implicit s: `AWS::EC2::Subnet`) = {
```

Comment:
Good catch! I had done some standing up of NATs using this code that seemed to work just fine without the depends. Perhaps I just got lucky, either way better safe than sorry.